### PR TITLE
Test to verify correct behavior with inconsistent Soundcloud reply

### PR DESCRIPTION
**Pull request by mikeger**
Relates to issue fixed in https://github.com/wireapp/wire-ios/pull/126

### CODE REVIEW

File: Wire-iOS.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -518,7 +518,7 @@
 		8786072F1D33E3FF00971A19 /* metamacros.h in Headers */ = {isa = PBXBuildFile; fileRef = 878607171D33E3FF00971A19 /* metamacros.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		878607301D33E3FF00971A19 /* weakify.h in Headers */ = {isa = PBXBuildFile; fileRef = 878607181D33E3FF00971A19 /* weakify.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		8787BDD91BF374420096B1B5 /* SettingsPropertyTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8787BDD61BF374420096B1B5 /* SettingsPropertyTests.swift */; };
-		878893641D49113600086646 /* JPSimulatorHackFramework.framework in CopyFiles */ = {isa = PBXBuildFile; fileRef = 878893621D490EB600086646 /* JPSimulatorHackFramework.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
+		878893641D49113600086646 /* (null) in CopyFiles */ = {isa = PBXBuildFile; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
```
Comment (daehn) [inline Wire-iOS.xcodeproj/project.pbxproj]: Can be removed, this looks like there is a reference to the `JPSimulatorHackFramework` somewhere in the project and Xcode does some weird things.

Comment (mikeger) [inline Wire-iOS.xcodeproj/project.pbxproj]: True, I removed it already on `develop`, let me clean it here

**Comment by mikeger** [comment]
@daehn please re-review

**Comment by mikeger** [comment]
`~# ping @daehn -n ` :100:

**Pull request merged**